DESCRIPTION

```diff
@@ -12,7 +12,10 @@ Description: The package provides convenient access to StatCan census data via C
 License: MIT
 Encoding: UTF-8
 LazyData: true
-Depends: dplyr (>= 0.7.2),  httr (>= 1.2.1), jsonlite (>= 1.5), digest (>= 0.6.12)
+Imports: digest (>= 0.1),
+    dplyr (>= 0.7.2),
```

Comment:
That's a good point, although I think now that the move to `rlang` is complete there will be fewer issues.